src/main/java/lotto/domain/Prize.java

```diff
@@ -0,0 +1,66 @@
+package lotto.domain;
+
+public enum Prize {
+    NOTHING(0L, 2, 999 ),
+    FIFTH(5000L, 3, 5),
+    FOURTH(50000L, 4, 4),
+    THIRD(1500000L, 5, 3),
+    SECOND(30000000L, 5, 2),
+    FIRST(2000000000L, 6, 1);
```

Comment:
3자리 수 마다 `_`을 넣어주면 값을 한눈에 파악할 수 있을 것 같아요. 😉
```suggestion
    FIRST(2_000_000_000L, 6, 1);
```